packages/oracle-manager/contracts/storage/NodeFactoryStorage.sol

```diff
@@ -0,0 +1,36 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract NodeFactoryStorage {
```

Comment:
note this will probably need to be updated very soon due to the impending router refactor stuff